### PR TITLE
Add information form interviews (requ. elicitation)

### PR DESCRIPTION
**Pull request by lesser1**
I added information from the latest two interviews with a) a researcher and b) external developers. For each interview I created a separate commit. For both interview partners I added an anonymized name in order to trace requirements back to them eventually.

### CODE REVIEW

File: docs/requirements/requirements.md [hunk 1]
```diff
@@ -205,22 +232,83 @@ This document is in the making since we are still in the requirements engineerin
...
+- Origin: Researcher *Toni*
+
+##### Open Question
+
+- Is this basically requirement R-Conf-003?
+
+#### R-Conf-005: System variants
+
+- The system should have different variants.
+- e.g., one variant with database as a service, another variant with a shared database
```
Comment (legendofa) [inline docs/requirements/requirements.md:329]: Which variants could be options except the database example?

File: docs/requirements/requirements.md [hunk 2]
```diff
@@ -205,22 +232,83 @@ This document is in the making since we are still in the requirements engineerin
...
+- The system must scale automatically.
+- Origin: Examiner, Researcher *Toni*
 
 ##### Open Question
 
 - What exactly does *elastic* mean? What is the examiner's understanding of the term?
 
+#### R-NFR-003: Resilience
+
+- The system should be resilient.
```
Comment (legendofa) [inline docs/requirements/requirements.md:287]: Is there a way we can be more specific about what circumstances the system should be able to withstand?

File: docs/requirements/requirements.md [hunk 3]
```diff
@@ -144,7 +175,9 @@ This document is in the making since we are still in the requirements engineerin
 - There must be a configurations file for designing the experiments. See R-U-003.
 - The RA must be sufficiently deterministic in order to allow the user to repeatedly carry out experiments.
 - Researchers want to do experiments using the RA.
-- Origin: Supervisors, Researchers
+- It must be effortless to use the system for experimentation. -- Researcher *Toni*
+- As a researcher, I want control over my experiments. The systems must allow me to have this control. -- Researcher *Toni*
```
Comment (legendofa) [inline docs/requirements/requirements.md]: Is this similar to the controlled environment mentioned elsewhere?

File: docs/requirements/requirements.md (hunk 1 above)
Comment (lesser1) [reply docs/requirements/requirements.md:329]: Good question, Idk right now. Of course, we could try to come up with something. The stakeholder who mentioned this only had this one example but he was insisting that variants would be great. I will add this as an open question to the requirement.

File: docs/requirements/requirements.md (hunk 2 above)
Comment (lesser1) [reply docs/requirements/requirements.md:287]: I think we have to conduct our stakeholders once more in order to clarify this. Right now, we are lacking detailed information to make more specific statement. The stakeholder said that the services should not disappear all of a sudden -- I can add that to the requirement.

File: docs/requirements/requirements.md (hunk 3 above)
Comment (lesser1) [reply docs/requirements/requirements.md]: Yes and no. The controlled environment refers to the researcher being able to control the state of the system at any experiment's start. This allows for repeatability. That means a researcher can be sure that the system is in the same state at the beginning of each experiment. Apart from that, the stakeholder -- researcher Toni -- also said that in general he wants to be able to control everything about an experiment, from start to finish. Therefore he requires a system that supports that and configuration unlocks this potential.

**Pull request merged**